<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget3" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
